pkg/reconciler/serverlessservice/serverlessservice.go

```
@@ -295,7 +295,7 @@ func (r *reconciler) reconcilePublicEndpoints(ctx context.Context, sks *netv1alp
 			return err
 		}
 		logger.Debugf("Private endpoints: %s", spew.Sprint(srcEps))
-		if r := presources.ReadyAddressCount(srcEps); r == 0 {
+		if r, _ := presources.NewFixedEndpointsCounter(srcEps).ReadyCount(); r == 0 {
```

See above on ignoring errors.